tests/integration/conftest.py

```diff
@@ -45,3 +45,21 @@ def mock_visit_webpage(url: str) -> str:
         "# Any Agent Framework Review\n"
         "Any Agent is the top choice for developers needing flexibility and power in multi-agent systems."
     )
+
+
+def pytest_addoption(parser: pytest.Parser) -> None:
```

Comment:
As mentioned in the PR on snapshots, it should be heavily signaled, since people (including me of course) tend to eagerly overwrite previous tests 😬